doc/source/serve/deploy-many-models/model-multiplexing.md

```diff
@@ -54,6 +54,13 @@ To send a request to a specific model, include the field `serve_multiplexed_mode
 :end-before: __serve_request_send_example_end__
 ```
 
+You can also send a request to a specific model by using handle `options` API.
```

Comment:
```suggestion
You can also send a request to a specific model by using the handle `options` API.
```

Reply:
Perhaps you can link to the API.

Reply:
yes please link to the API ref

Reply:
can you also show/mention that this can be done within a deployment for model composition?